### PR TITLE
Add ion-sound v3.0.7

### LINKED ISSUE #7340: [Request] ion.sound
**Library name: ion.sound** 
**Git repository url: https://github.com/IonDen/ion.sound**
**npm package url(optional): npm install ion-sound** 
**License(s):**
**Official homepage: http://ionden.com/a/plugins/ion.sound/en.html**
**Wanna say something? Leave message here: The guy who makes this speaks english**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by x09326**
@PeterDaveHello please help me check my PR for #7340 , thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/IonDen/ion.sound
 * Official website (optional, not the repository): http://ionden.com/a/plugins/ion.sound/en.html
 * NPM package url (optional): https://www.npmjs.com/package/ion-sound
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 481
   - Count of watchers: 37
   - Count of forks: 92
 * NPM download stats (optional):
   - Downloads in the last day: 20
   - Downloads in the last week: 34
   - Downloads in the last month: 155

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

with npm auto-update config added close #7340 cc @rgbjoy

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
According to its document, we'll need the `sounds/` folder.

http://ionden.com/a/plugins/ion.sound/en.html

**Comment by x09326** [comment]
@PeterDaveHello I added the sounds folder and modified the npm auto-update
Please help me check it, thank you.

File: ajax/libs/ion-sound/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "ion-sound",
+  "filename": "ion.sound.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/ion-sound/package.json]: `"filename": "js/ion.sound.min.js",` ?

**Comment by x09326** [comment]
@PeterDaveHello Please help me check it, I modified the filename.
Thank you.

**Pull request merged**